src/preprocessing/passes/ackermann.h

```diff
@@ -61,9 +65,18 @@ class Ackermann : public PreprocessingPass
  private:
   /* Map each function to a set of terms associated with it */
   FunctionToArgsMap d_funcToArgs;
-  /* Map each function term to the new Skolem variable created by
+  /* Map each function-application term to the new Skolem variable created by
    * ackermannization */
   theory::SubstitutionMap d_funcToSkolem;
+  /* Map each uninterpreted sort term to the new Skolem variable created by
+   * ackermannization */
+  theory::SubstitutionMap d_sortsToSkolem;
```

Comment:
We don't map sorts to skolems, but variables of uninterpreted sorts to variables of bv sorts. So I suggest renaming this field to something like
`d_usVarsToBVVars` (or something nicer with the same meaning).